website/content/docs/nia/cli/start.mdx

```diff
@@ -0,0 +1,62 @@
+---
+layout: docs
+page_title: Start Command
+description: >-
+  Consul-Terraform-Sync supports start command for running CTS in daemon mode
+---
+
+# start
+
+The start command runs Consul-Terraform-Sync (CTS) in Daemon mode. When running as a daemon, there is no default configuration to run CTS in a meaningful way. Setting a configuration flag `-config-file` or `-config-dir` is required. For example:
```

Comment:
```suggestion
The `start` command starts Consul-Terraform-Sync (CTS) in daemon mode. When running as a daemon, there is no default configuration to start CTS in a meaningful way. Setting a configuration flag `-config-file` or `-config-dir` is required. For example:
```

Feel free to take / leave ^. The snippet suggests
 - adding "``" around "start" in "start command" - aligns with other command docs
 - lower casing "d" in "Daemon mode"
 - switches to using "start" language to align with name of command

Reply:
General suggestion to search on "run" and replace (where it makes sense) with "start" to add alignment with name of command

Reply:
that's a good point, I'll do that